dev/Pager/PrototypePager/PrototypePager.Events.cs

```diff
@@ -0,0 +1,185 @@
+﻿using Microsoft.UI.Xaml.Controls;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Automation.Provider;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+
+namespace MUXControlsTestApp
+{
+    public sealed partial class PrototypePager : Control
+    { 
+
+        private static void OnPagerDisplayModeChanged(DependencyObject sender, DependencyPropertyChangedEventArgs args)
+        {
+            PagerDisplayModes mode = (PagerDisplayModes)args.NewValue;
+            PrototypePager pager = sender as PrototypePager;
+            switch (mode)
+            {
+                case PagerDisplayModes.NumberBox:
+                    VisualStateManager.GoToState(pager, "NumberBoxVisible", false);
```

Comment:
just use `this` instead

Reply:
same for the rest of these as well.

---
In reply to: [463293391](https://github.com/microsoft/microsoft-ui-xaml/pull/2977#discussion_r463293391) [](ancestors = 463293391)

Reply:
Done.